packages/desktop/views/dashboard/wallet/views/multi-step/OneTimeDeposit.svelte

```diff
@@ -2,18 +2,19 @@
     import { Button, FontWeight, Text, TextType } from 'shared/components'
     import { localize } from '@core/i18n'
     import { setClipboard } from '@core/utils'
+    import { selectedWallet } from '@core/wallet'
 
     let isAddressShown: boolean = false
+    let depositAddress: string = ''
 
-    // TODO: Replace with proper address
-    const DUMMY_ADDRESS = 'iota1q9f0mlq8yxpx2nck8a0slxnzr4ef2ek8f5gqxlzd0wasgp73utryjtzcp98'
+    $: depositAddress = $selectedWallet?.depositAddress
```

Comment:
the one time deposit address is not the deposit address but the `implicit account address`